staging/src/k8s.io/apimachinery/pkg/api/meta/help.go

```diff
@@ -221,6 +245,9 @@ func extractList(obj runtime.Object, allocNew bool) ([]runtime.Object, error) {
 	if err != nil {
 		return nil, err
 	}
+	if items.IsNil() {
```

Comment:
Do we need another test in `TestStreamingListEncode` for this case?

Reply:
It's covered in the first case for empty.

Reply:
thanks!